enarx-keep-sgx-shim/src/start.S

```diff
@@ -156,13 +184,15 @@ enclave:
 syscall:
     savep                                           # Save preserved registers
     mov     %rsp,                   SRSP(%rcx)      # Save restoration stack pointer
+    
+    zerox                                           # Clear xCPU state
+    zerof   %rcx                                    # Clear CPU flags
+    xor     %rcx,                   %rcx            # Clear %rcx
```

Comment:
```suggestion
    xor     %rcx,                   %rcx            # Clear %rcx
    zerof   %rcx                                    # Clear CPU flags
    zerox                                           # Clear xCPU state
```

Reply:
See the documentation for `zerof` for why.

Reply:
`%rcx` must be zero ... got it.